doc/classes/GraphNode.xml

```diff
@@ -276,6 +276,11 @@
 		</member>
 	</members>
 	<signals>
+		<signal name="slot_sizes_changed">
+			<description>
+				Emitted when any slot's size may have changed.
```

Comment:
```suggestion
				Emitted when any slot's size might have changed.
```
"May" is ambiguous, and is considered by some to be invalid English, "might" is safer for "possibly" as opposed to the meaning of "may" as in "is permitted"